### DISCUSSION TITLE
Add support for service return values

### DISCUSSION THREAD

**Discussion by ScottG489** [zachowj/node-red-contrib-home-assistant-websocket#961] [Feature Request]
Coming up in the next release of HA ([currently already available in beta](https://rc.home-assistant.io/blog/2023/06/28/release-20237/#services-call-can-now-respond)), services will be able to return data. More information here:

https://github.com/home-assistant/architecture/discussions/777

I think it would make sense for the [call service node](https://zachowj.github.io/node-red-contrib-home-assistant-websocket/node/call-service.html) to support returning this as part of it's output.

**Comment by zachowj** [comment]
https://github.com/zachowj/node-red-contrib-home-assistant-websocket/issues/972

**Comment by zachowj** [comment]
Responses were added to `call_service` endpoint. 

https://github.com/home-assistant/core/commit/618b666126a885bfb0ae44571bec0c44ff5bdada
